src/find_zero.jl

```diff
@@ -166,22 +166,85 @@ log_nmethod(::NullTracks, method) = nothing
 # a tracks object to record tracks
 """
     Tracks(T, S)
+    Tracks()
 
 Construct a `Tracks` object used to record the progress of the algorithm.
+`T` is the type of function inputs, and `S` is the type of function outputs. They 
+both default to `Float64`. Note that because this type is not exported, you have to 
+write `Roots.Tracks()` to construct a `Tracks` object.
 
-By default, a null tracks object used, but if `verbose=true` is specified of a `Tracks` object passed to the keyword `tracks` then the steps of the algorithm are recorded in the `Tracks` object. If `verbose=true`, the `Tracks` object will be shown.
+By default, no tracking is done while finding a root (with `find_zero`, `solve`, or `fzero`). 
+To change this, construct a `Tracks` object, and pass it to the keyword argument `tracks`. 
+This will modify the `Tracks` object, storing the input and function values at each iteration, 
+along with additional information about the root-finding process.
 
-Alternatively, a `Tracks` object can be constructed and then shown; `find_zero` modifies a `Tracks` object.
+`Tracker` objects are printed in a nice, easy-to-read format. You can also 
+access individual fields if you need the values. A `Tracks` object has the following fields:
```

Comment:
As there is a chance this could be refactored, it would be best to include the usual reminder that internal fields are subject to modification. (Maybe just say "Currently a `tracks` object ...")

Reply:
So "Currently a `tracks` object is printed in a nice, easy-to-read format..."?

Reply:
I was referring to the fieldname (tracks.xs, tracks.fs...) These may change, though unlikely, so I'm suggesting using "currently" at the point you describe these.

Reply:
I would say that the documentation should be updated if the field names change. The docs should not be uncertain about what things are called.

Reply:
Once documented, even if not exported, they become expected by users, making it harder to modify things should a refactoring be done. (This is exactly why I'm reluctant to rename `Tracks`, even if I agree with your take). Take for example the discussion on the need to two types of ways to record `xs` (one with brackets, one without). I'd like the freedom to change that without feeling responsible for mucking up someone's code.

Reply:
Yes, for sure. But changing things on an old package is usually best reserved for necessary things (performance improvements...)

Reply:
I am having a hard time seing exactly what you want to change. Feel free to make the change yourself - I trust your judgment